src/components/header_menu/HeaderMenuToolbar.jsx

```diff
@@ -215,19 +216,23 @@ class HeaderMenuToolbar extends React.Component {
                     </ListItemIcon>
                     Boundary Conditions
                 </MenuItem>
-                <MenuItem disabled>
+                <MenuItem disabled hidden>
                     <ListItemIcon>
                         <PolymerIcon />
                     </ListItemIcon>
                     Polymer
                 </MenuItem>
-                <MenuItem disabled>
+                <MenuItem disabled hidden>
                     <ListItemIcon>
                         <NanotubeIcon />
                     </ListItemIcon>
                     Nanotube
                 </MenuItem>
-                <MenuItem onClick={() => this.setState({ showPythonTransformation: true })}>
+                <MenuItem
+                    disabled
+                    hidden
```

Comment:
@VsevolodX - I disabled pyodide option in dev until we have it working. We can start an epic branch if needed to move the work forward.

Reply:
I'll look into it, but most likely my current task will be enough to enable back